### PR TITLE
Prepare website structure for 4.8.1 release - DO NOT MERGE

### PR DESCRIPTION
**Pull request by eolivelli**
Signed-off-by: eolivelli <eolivelli@apache.org>

Descriptions of the changes in this PR:

### Motivation

Prepare the wesite for 4.8.1 release

### Changes

Create the structure for docs for 4.8.1

> ---
> In order to uphold a high standard for quality for code contributions, Apache BookKeeper runs various precommit
> checks for pull requests. A pull request can only be merged when it passes precommit checks. However running all
> the precommit checks can take a long time, some trivial changes don't need to run all the precommit checks. You
> can check following list to skip the tests that don't need to run for your pull request. Leave them unchecked if
> you are not sure, committers will help you:
>
> - [x] [skip bookkeeper-server bookie tests]: skip testing `org.apache.bookkeeper.bookie` in bookkeeper-server module.
> - [x] [skip bookkeeper-server client tests]: skip testing `org.apache.bookkeeper.client` in bookkeeper-server module.
> - [x] [skip bookkeeper-server replication tests]: skip testing `org.apache.bookkeeper.replication` in bookkeeper-server module.
> - [x] [skip bookkeeper-server tls tests]: skip testing `org.apache.bookkeeper.tls` in bookkeeper-server module.
> - [x] [skip bookkeeper-server remaining tests]: skip testing all other tests in bookkeeper-server module.
> - [x] [skip integration tests]: skip docker based integration tests. if you make java code changes, you shouldn't skip integration tests.
> - [x] [skip build java8]: skip build on java8. *ONLY* skip this when *ONLY* changing files under documentation under `site`.
> - [x] [skip build java9]: skip build on java9. *ONLY* skip this when *ONLY* changing files under documentation under `site`.
> ---

> ---
> Be sure to do all of the following to help us incorporate your contribution
> quickly and easily:
>
> If this PR is a BookKeeper Proposal (BP):
>
> - [ ] Make sure the PR title is formatted like:
>     `<BP-#>: Description of bookkeeper proposal`
>     `e.g. BP-1: 64 bits ledger is support`
> - [ ] Attach the master issue link in the description of this PR.
> - [ ] Attach the google doc link if the BP is written in Google Doc.
>
> Otherwise:
> 
> - [ ] Make sure the PR title is formatted like:
>     `<Issue #>: Description of pull request`
>     `e.g. Issue 123: Description ...`
> - [ ] Make sure tests pass via `mvn clean apache-rat:check install spotbugs:check`.
> - [ ] Replace `<Issue #>` in the title with the actual Issue number.
> 
> ---

### CODE REVIEW

File: site/releases.md [hunk 1]
```diff
@@ -64,6 +64,10 @@ Client Guide | API docs
 
 ## News
 
+### [date] Release {{ site.latest_version }} available
```
Comment (sijie) [inline site/releases.md]: remove this

File: site/_config.yml [hunk 2]
```diff
@@ -32,7 +33,7 @@ archived_versions:
 - "4.1.0"
 - "4.0.0"
 latest_version: "4.9.0-SNAPSHOT"
-latest_release: "4.8.0"
+latest_release: "4.8.1"
```
Comment (sijie) [inline site/_config.yml]: undo this change. so the version is not shown up at the menu.

Comment (eolivelli) [reply site/_config.yml]: My idea was not to commit this PR but only the final one.
Anyway I am fine with this approach, changing this PR, I will rebase the second one

Comment (eolivelli) [reply site/_config.yml]: done

File: site/releases.md (hunk 1 above)
Comment (eolivelli) [reply site/releases.md]: done

File: site/_config.yml [hunk 3]
```diff
@@ -8,6 +8,7 @@ destination: local-generated
 twitter_url: https://twitter.com/asfbookkeeper
 
 versions:
+- "4.8.1"
```
Comment (sijie) [inline site/_config.yml]: this also need to be reverted.
